huntsman/tests/test_huntsman.py

```diff
@@ -288,7 +290,7 @@ def start_pocs():
         assert wait_for_running(msg_subscriber)
         observatory.logger.info('Got RUNNING message')
 
-        time.sleep(5)
+        time.sleep(2)
```

Comment:
What's reason for making this less?  If anything I would suspect the test is hanging because this sleep time is too short. 

In general though, if these changes to the test didn't fix the test and then we are skipping the test anyway, I would prefer not to change the existing code unless we know that actually accomplishes something.  In theory it was working at some point with the code as is, so I wouldn't want to complicate the debugging.

Reply:
Again, just copying something from POCS in a desperate attempt to fix the test.  I agree though, since these didn't work I should take these back out.